integrated_channels/moodle/migrations/0031_auto_20231204_0737.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.20 on 2023-12-04 07:37
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('moodle', '0030_merge_0028_auto_20231116_1826_0029_auto_20231106_1233'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='historicalmoodleenterprisecustomerconfiguration',
+            name='use_encrypted_user_data',
+            field=models.BooleanField(default=False, help_text='When set to True, the configured customer will use encrypted columns data to make client requests, this a boolean flag for testing purpose'),
+        ),
+        migrations.AddField(
+            model_name='moodleenterprisecustomerconfiguration',
+            name='use_encrypted_user_data',
+            field=models.BooleanField(default=False, help_text='When set to True, the configured customer will use encrypted columns data to make client requests, this a boolean flag for testing purpose'),
+        ),
+    ]
```

Comment:
[Question] These fields are added here but are being removed in the second migration?

Reply:
@sameenfatima78 this was added when I was using the feature flag by keeping it inside the config model.
should be removed.